source/MRMesh/MRExtractIsolines.cpp

```diff
@@ -55,6 +62,54 @@ std::vector<std::vector<MeshEdgePoint>> Isoliner::extract()
     return res;
 }
 
+IsoLine Isoliner::track( const MeshTriPoint& start, ContinueTrack continueTrack )
+{
+    auto testEdge = [&] ( EdgeId e )->std::optional<MeshEdgePoint>
+    {
+        VertId o = topology_.org( e );
+        VertId d = topology_.dest( e );
+        float vo = valueInVertex_( o );
+        float vd = valueInVertex_( d );
+        if ( vd < 0 && 0 <= vo )
+            return  toEdgePoint_( e.sym(), vd, vo );
```

Comment:
This only tests that the plane intersects given edge, but do we go in correct direction?